### ISSUE TITLE
a11y

### ISSUE THREAD

**Issue by kueda** [inaturalist/iNaturalistReactNative#2123]
There's a lot we could do to make the app more accessible for hard launch, but we're going to focus on these areas.

* screenreaders
* support larger text sizes

# screenreaders
## overall approach

* we want to target a few key flows
* assess with [Accessibility Inspector](https://developer.apple.com/documentation/accessibility/inspecting-the-accessibility-of-screens)
* click test using VoiceOver (we should be able to accomplish those flows with just VoiceOver)

## research
* what kind of automated tools could we use to audit accessibility
* can we find a few people who use screenreaders and observe how they use iNat (record media in app vs import, do they use a screen reader, do they navigate by touch or just by elements being read)
* accessibility unit tests: we already have a few, is it practical to use them more widely?
* is https://github.com/google/GSCXScanner worth using?

## key flows
  * creating a new observation by recording a new sound
  * log in sign up
  * upload
  * getting to one of your observations from MyObs

## todo
- [ ] @abhasinat: find three people who use screenreaders who would be willing to
  1. Tell us how they use iNat
  2. Try iNat Next and tell us where they get stuck
- [ ] spend a week trying to write accessibility unit tests using the library we're currently using
- [ ] allow obs creation w/ sound to work w/ VoiceOver
- [ ] allow login and sign up to work w/ VoiceOver
- [ ] allow MyObs to provide access to ObsDetails and support upload w/ VoiceOver

# larger text
